logic/src/commonTest/kotlin/com/wire/kalium/logic/data/connection/ConnectionRepositoryTest.kt

```diff
@@ -72,6 +74,34 @@ class ConnectionRepositoryTest {
         result.shouldSucceed()
     }
 
+    @Test
+    fun givenAConnectionRequest_WhenSendingAConnection_thenTheConnectionShouldBeSentAndPersistedLocally() = runTest {
+        // given
+        val userId = UserId("user_id", "domain_id")
+        given(connectionApi)
+            .suspendFunction(connectionApi::createConnection)
+            .whenInvokedWith(eq(userId))
+            .then { NetworkResponse.Success(connection1, mapOf(), 200) }
+        given(conversationDAO)
+            .suspendFunction(conversationDAO::insertOrUpdateOneOnOneMemberWithConnectionStatus)
+            .whenInvokedWith(eq(UserIDEntity(userId.value, userId.domain)), any(), any())
+            .then { _, _, _ -> return@then }
+
+        // when
+        val result = connectionRepository.sendUserConnection(com.wire.kalium.logic.data.user.UserId(userId.value, userId.domain))
```

Comment:
maybe we could add `com.wire.kalium.logic.data.user.UserId` as a proper import or with a different name in case there is already one with the same name 🤔

Reply:
Good idea, I did some changes to have it imported with an alias